app/messages/service_credential_key_binding_create_message.rb

```diff
@@ -0,0 +1,13 @@
+module VCAP::CloudController
+  class ServiceCredentialKeyBindingCreateMessage < ServiceCredentialBindingCreateMessage
+    validates :name, string: true, presence: true
+
+    def relationships_message
+      @relationships_message ||= Relationships.new(relationships&.deep_symbolize_keys)
+    end
+
+    class Relationships < ServiceCredentialBindingCreateMessage::Relationships
+      validates_with NoAdditionalKeysValidator
```

Comment:
I don't think we gain much from inheriting here. For me it makes the code more difficult to follow and we are just reusing a couple of configuration lines.